compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropView.skiko.kt

```diff
@@ -16,6 +16,51 @@
 
 package androidx.compose.ui.viewinterop
 
-// Interop Views aren't supported in Skiko. Use Any instead of Nothing because we can't have a type
-// aliased to Nothing appear as a function return type or property type.
-actual typealias InteropView = Any
\ No newline at end of file
+import androidx.compose.ui.InternalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.PointerEvent
+import androidx.compose.ui.input.pointer.PointerEventPass
+import androidx.compose.ui.input.pointer.PointerInputFilter
+import androidx.compose.ui.node.ModifierNodeElement
+import androidx.compose.ui.node.PointerInputModifierNode
+import androidx.compose.ui.unit.IntSize
+
+/**
+ * Modifier to associate an [InteropView] with a [PointerInputFilter] to allow hit testing it and
+ * perform custom pointer input handling if needed.
+ */
+@InternalComposeUiApi
+open class InteropViewModifierNode(
+    var interopView: InteropView
+) : Modifier.Node(), PointerInputModifierNode {
+    override fun onPointerEvent(
+        pointerEvent: PointerEvent,
+        pass: PointerEventPass,
+        bounds: IntSize
+    ) {}
+
+    override fun onCancelPointerInput() {}
+}
+
+/**
+ * Element for [InteropViewModifierNode]. A custom implementation of [ModifierNodeElement] is needed
+ * for possible [InteropViewModifierNode] subclasses.
+ */
+internal data class InteropViewModifierNodeElement(
+    val interopView: InteropView
+) : ModifierNodeElement<InteropViewModifierNode>() {
+    override fun create(): InteropViewModifierNode =
+        InteropViewModifierNode(interopView)
+
+    override fun update(node: InteropViewModifierNode) {
+        node.interopView = interopView
+    }
+}
+
+/**
+ * Add an association with [InteropView] to the modified element.
+ * Allows hit testing and custom pointer input handling for the [InteropView].
+ */
+@InternalComposeUiApi
+fun Modifier.interopView(view: InteropView): Modifier =
```

Comment:
The name is too generic. I assume it will show up in suggestions for unrelated cases

Reply:
modifier is no a VIEW. So at least something like `interopViewAnchor`.

Wait. We don't need this function in public at all - let's just keep a node. (InteropView**Anchor**ModifierNode)